src/engine/RuntimeInformation.h

```diff
@@ -109,7 +111,8 @@ class RuntimeInformation {
   // The details of the LIMIT/OFFSET, the time (in ms) that was spent computing
   // it, and the information whether the `actual` operation (the old root of the
   // runtime information) is written to the cache, are passed in as arguments.
-  void addLimitOffsetRow(const LimitOffsetClause& l, size_t timeForLimit,
+  void addLimitOffsetRow(const LimitOffsetClause& l,
+                         std::chrono::milliseconds timeForLimit,
```

Comment:
Maybe for readability you can also introduce `RuntimeInformation::Milliseconds` for shorter lines.